webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,171 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Grid,
+  Tab,
+  Tabs,
+  GridItem,
+  Badge,
+  Title,
+  Breadcrumb,
+  BreadcrumbItem,
+  Text,
+  TextVariants,
+  PageSection,
+  PageSectionVariants,
+} from '@patternfly/react-core';
+
+import Skeleton from 'react-loading-skeleton';
+import RelativeDateTime from '../../components/common/dates/RelativeDateTime';
+import StatusAlert from './Status';
+
+import { foremanUrl } from '../../../foreman_tools';
+import { get } from '../../redux/API';
+import { selectAPIResponse } from '../../redux/API/APISelectors';
+
+import Properties from './Properties';
+import ParametersCard from './Parameters';
+import InterfacesCard from './Interfaces';
+import AuditCard from './Audits';
+import ActionsBar from './ActionsBar';
+
+import './HostDetails.scss';
+
+const HostDetails = ({ match }) => {
+  const dispatch = useDispatch();
+  const [activeTab, setActiveTab] = useState(0);
+  const response = useSelector(state =>
+    selectAPIResponse(state, 'HOST_DETAILS')
+  );
+  useEffect(() => {
+    dispatch(
+      get({
+        key: 'HOST_DETAILS',
+        url: foremanUrl(`/api/hosts/${match.params.id}`),
+      })
+    );
+  }, [match.params.id, dispatch]);
+
+  useEffect(() => {
+    document.body.classList.add('transprenet-border');
+  }, []);
+
+  const handleTabClick = (event, tabIndex) => {
+    setActiveTab(tabIndex);
+  };
+  return (
+    <>
+      <PageSection className="header" variant={PageSectionVariants.light}>
+        <div style={{ marginLeft: '18px', marginRight: '18px' }}>
+          <Breadcrumb style={{ marginTop: '15px' }}>
+            <BreadcrumbItem to="/hosts">Hosts</BreadcrumbItem>
+            <BreadcrumbItem isActive>
+              {response.name || <Skeleton />}
+            </BreadcrumbItem>
+          </Breadcrumb>
+          {/* TODO: Replace all br with css */}
+          <br />
+          <br />
+          <Grid>
+            <GridItem span={2}>
+              <Title headingLevel="h5" size="2xl">
```

Comment:
Title can become really big, can you add ellipsis with tooltip to it? so it would still be a one liner?
![Selection_336](https://user-images.githubusercontent.com/26363699/87457990-f1b2ec00-c611-11ea-9190-bc5128dfe2a3.png)

Reply:
I'm not sure if  ellipsis fit titles from UX perspective, we have lots of space to keep it as a one-line (just a css issue)